src/lp.jl

```diff
@@ -0,0 +1,174 @@
+#=
+Interface to lrs_solve_lp
+
+=#
+
+
+mutable struct LRSLinprogSolution
+    status::Symbol  # :Optimal, :Infeasible, :Unbounded, :UserLimit, or :Error
+    objval::Union{Nothing,Rational{BigInt}}
+    sol::Vector{Rational{BigInt}}
+    attrs::Dict
+end
+
+
+function setobj(P::Ptr{Clrs_dic}, Q::Ptr{Clrs_dat},
+                c::Vector{<:Real}, maximize::Bool)
+    row = Vector{Rational{BigInt}}(undef, unsafe_load(P).d+1)
+    sgn = maximize ? 1 : -1
+    field = maximize ? :maximize : :minimize
+    row[1] = 0
+    row[2:end] = c * sgn
```

Comment:
I would do `= maximize ? sgn : -sgn`. If `maximize` is `true` it will not do any multiplication which is more efficient